### ISSUE TITLE
APK3 | Crea Evento -> N° max Fitfellas - Tasto Conferma non si attiva 

### ISSUE THREAD

**Issue by fitfella** [LorenzoMazz/whereverFitness#70]
il tasto "Conferma" non si attiva (i.e.rimane grigio) se non si inserisce il N° max dei fitfella che dovrebbe essere un campo di testo non obbligatario

**Comment by riki1214** [comment]
Vero, fatto. Ho comunque lasciato la descrizione obbligatoria. Se non fosse obbligatoria neanche quella scrivetemelo qui sotto.

**Comment by fitfella** [comment]
Se compare la scritta rossa quando il campo "descrizione" rimane vuoto è corretto.
